packages/ui/src/Social.tsx

```diff
@@ -13,14 +13,15 @@ type SocialProps = {
 }
 const TOUCH_TARGET: TouchTarget = {
   touchAreaPadding: 'p-2',
-  touchAreaOffset: '-m-2 sm:mx-0',
+  touchAreaOffset: '-m-2',
```

Comment:
Any changes that we make to this component will also apply to FF. It's a shared component. By changing this, we break the mobile nav when `Social` is used.